RefRed/export/export_plot_ascii.py

```diff
@@ -58,30 +60,32 @@ def export_ix(self):
         [row, col] = self.get_current_row_col_displayed()
         _data = big_table_data[row, col]
         run_number = _data.run_number
-        default_filename = 'REFL_' + run_number + '_ix.txt'
+        default_filename = "REFL_" + run_number + "_ix.txt"
         path = parent.path_ascii
-        default_filename = path + '/' + default_filename
+        default_filename = path + "/" + default_filename
```

Comment:
The "better" way is to use [pathlib](https://docs.python.org/3.6/library/pathlib.html) which would allow this to be (assuming imports are already done)
```python
path = Path(parent.path_ascii)
default_filename = path + default_filename
```
Easy enough to fix